src/Asset.js

```diff
@@ -73,6 +73,11 @@ class Asset {
     let resolved = path
       .resolve(path.dirname(from), url)
       .replace(/[\?#].*$/, '');
+
+    if (!fs.existsSync(resolved)) {
```

Comment:
mmm, rather not be doing sync file io here. also, this doesn't take into account that the resolver may do additional steps to find the path (e.g. checking file extensions, aliases, etc.). finally, it will prevent errors from occurring in legitimate cases where the file should really be there.

let's think about how to solve this in a different way.